Xamarin.Forms.Core/FontAttributes.cs

```diff
@@ -3,10 +3,48 @@
 namespace Xamarin.Forms
 {
 	[Flags]
+	[TypeConverter(typeof(FontAttributesConverter))]
 	public enum FontAttributes
 	{
 		None = 0,
 		Bold = 1 << 0,
 		Italic = 1 << 1
 	}
+
+	[Xaml.TypeConversion(typeof(FontAttributes))]
+	public sealed class FontAttributesConverter : TypeConverter
+	{
+
+		public override object ConvertFromInvariantString(string value)
+		{
+			if (string.IsNullOrEmpty(value))
+				return FontAttributes.None;
+
+			FontAttributes attributes = FontAttributes.None;
+			value = value.Trim();
+			if (value.Contains(",")) { //Xaml
+				foreach (var part in value.Split(','))
```

Comment:
Should we still stick with not using foreach? Something like this

```C#
			string[] parts = null;
			if (value.Contains(","))
				parts = value.Split(',');
			else
				parts = value.Split(' ');

			for (int i = 0; i < parts.Length; i++)
				attributes |= ParseSingleAttribute(parts[i], value);
```

Reply:
I dont care in this case because its not a hot path

Reply:
Up to you to decide what you want

Reply:
I'd prefer it because
-consistency
-I like the one for loop applying that logic opposed to the duplication of the for loop

Reply:
hot path or not, `foreach` on an array is equally (well, even slightly more) efficient than a `for` loop... https://sharplab.io/#v2:C4LglgNgNAJiDUAfAAgBgATIIwG4CwAUMgMyYBM6AwugN6HoOanIAs6AsgBQCUt9jAgG4BDAE7oADmOABndAF50AIgBmAezXoARmKUA6AMoSIYYJwDk6c93wEBA9aICmwgMYALdJxHiJ6MAB2ktIy3Pz29tgAnJwSNuH2jl4+/vKoOGAAPFKisnoAMk4BAObA7hnw8GF2EZFYMTmyANpgALrxNQwAvoRdQA=

Reply:
That's fine and not something to hold up the PR but I still feel being consistently consistent is more important. If the code base uses *for* everywhere that's not an IEnumerable that's an indicator to contributors. That's something we can put in a coding guideline.  If we just seemingly randomly have foreach vs for then having any guideline about which to use seems pointless and either is always fine